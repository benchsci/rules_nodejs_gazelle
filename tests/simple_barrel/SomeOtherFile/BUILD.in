load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

# gazelle:js_collect_barrels

ts_project(
    name = "index",
    srcs = [
        "index.ts",
    ],
)

ts_project(
    name = "SomeOtherFile",
    srcs = [
        "SomeOtherFile.tsx",
    ],
)
