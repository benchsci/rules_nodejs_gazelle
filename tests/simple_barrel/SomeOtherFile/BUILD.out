load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

# gazelle:js_collect_barrels

ts_project(
    name = "SomeOtherFile",
    srcs = [
        "SomeOtherFile.ts",
        "index.ts",
    ],
    tags = ["js_barrel"],
)
